galaxy_ng/app/api/ui/views/auth.py

```diff
@@ -44,6 +46,10 @@ def get(self, request, *args, **kwargs):
 
     def post(self, request, *args, **kwargs):
         serializer = self.get_serializer(data=self.request.data)
+
+        if str(request.user) == "AnonymousUser" and GALAXY_ENABLE_VIEW_ONLY_ACCESS:
```

Comment:
Here it is also possible to read it from django settings

```suggestion
        if str(request.user) == "AnonymousUser" and settings.GALAXY_ENABLE_VIEW_ONLY_ACCESS:
```